src/test/java/edu/hawaii/its/api/service/PathFilterTest.java

```diff
@@ -0,0 +1,95 @@
+package edu.hawaii.its.api.service;
+
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+import static edu.hawaii.its.api.service.PathFilter.onlyMembershipPaths;
+import static edu.hawaii.its.api.service.PathFilter.pathHasBasis;
+import static edu.hawaii.its.api.service.PathFilter.pathHasExclude;
+import static edu.hawaii.its.api.service.PathFilter.pathHasInclude;
+import static edu.hawaii.its.api.service.PathFilter.pathHasOwner;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class PathFilterTest {
+
+    private String includePath = "bogus:include";
+    private String excludePath = "bogus:exclude";
+    private String ownerPath = "bogus:owners";
+    private String basisPath = "bogus:basis";
+
+    private List<String> testPathList;
+
+    @Test
+    public void testPathHasInclude() {
+        testPathList = new ArrayList<>();
+        assertEquals(filterPaths(testPathList, pathHasInclude()).size(), 0);
+        testPathList.add(includePath);
+        testPathList.add(includePath);
+        testPathList.add(excludePath);
+        testPathList.add(ownerPath);
+        testPathList.add(basisPath);
+        assertEquals(filterPaths(testPathList, pathHasInclude()).size(), 2);
```

Comment:
This is not a big deal, but for the future, the first param of `assertEquals` is the expected value...
```
assertEquals)(int expected, int actual);
Asserts that expected and actual are equal.
```
So if your test fails right now its gonna read...
```
expect: <whatever the value of filterPaths(testPathList, pathHasInclude()).size() is>
actual: 0
```
This can be confusing to other developers and yourself.